ui/readme.de.md

```diff
@@ -198,3 +198,22 @@ Dieses Icon markiert Such-Anfragen deren Ergebnis keinerlei Kunden-Interaktion a
 | Correctness    | Dieser Wert ist ein Maß für die linguistische und kontextuelle Korrektheit der Suchanfrage und damit das Produkt aus WordValidation und LMProbability (s.u.).                                                                                                                                                     |
 | WordValidation | Dieser Wert ist ein Maß für die linguistische Korrektheit der einzelnen Terme einer Suchanfrage. Sind sämtliche Terme einer Suchanfrage linguistisch korrekt ist der Score = 1.0                                                                                                                                  |
 | LMProbability  | Die kontextuelle Korrektheit einer Suchanfrage wird durch die language model probability ermittelt. Beispiel: Die Suchanfrage "damen herren" ist linguistisch korrekt und erhält somit einen WordValidationsScore von 1.0. Aber die LM-Probability ist 0.0 da die beiden Terme in Verbindung keinen Sinn ergeben. |
+
+## Clusteransicht - Clusterfilter
+| Filter	| Erklärung
+|----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| 
+| Ohne Master-Query	| Cluster, die aufgrund des mangelnden Vertrauens von searchHub in die Auswahl einer Master-Query nicht exportiert wurden. |
+| Manuell überstellte Master-Query	| Cluster, die eine vom Benutzer explizit ausgewählte Master-Query aufweisen, im Gegensatz zur Neigung von searchHub zu einer anderen auf Basis von Key Performance Indicators (KPIs). |
+| Fusioniert	| Cluster, die konsolidiert oder kombiniert wurden. |
+| Nicht fusioniert	| Cluster, die nicht konsolidiert oder nicht kombiniert wurden. |
+| Manuell ausgewählte Master-Query	| Cluster mit manuell ausgewählter Master-Query, unabhängig davon, ob das System dieselbe oder eine andere Master-Query auswählen würde. |
+| Manuell hinzugefügte Abfrage	| Cluster, die manuell eingefügte Abfragen enthalten. |
+| Mit Kampagnenabfrage	| Cluster mit als kuratiert markierten Abfragen |
+| Mit Suggest-Whitelist-Abfrage	| Cluster, in denen das Suggest-Label von einem Benutzer hinzugefügt oder geändert wurde und sich von der automatisch ausgewählten Bezeichnung unterscheidet. |
```

Comment:
Mit Suggest-Whitelist-Abfrage -> Mit Suggest-Whitelist-Query
Cluster, in denen das Suggest-Label manuell hinzugefügt oder geändert wurde und sich von dem automatisch ausgewählten Suggest-Label unterscheidet.